examples/quickstart/hello-world-application.yaml

```diff
@@ -19,9 +19,9 @@ spec:
         image: ghcr.io/wasmcloud/components/http-hello-world-rust:0.1.0
       traits:
         # Govern the spread/scheduling of the component
-        - type: spreadscaler
+        - type: daemonscaler
           properties:
-            replicas: 1
+            replicas: 100
```

Comment:
99% positive setting `replicas` is a noop for daemonscaler, but doesn't hurt

Reply:
We just use that number across all available instances, in contrast to spread where we only place the number you specify. So it's good!